javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,8 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words, suffix) {
+  var arr = [];
+  for (var i = 0; i < words.length; i++) {
+    arr[i] = words[i] + suffix;
```

Comment:
Instead of assigning this directly to an index, you should use the `push` method